src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -170,6 +174,15 @@ public static void AddCommonServices(IServiceCollection services)
             // specialized app.
             services.AddSingleton<IHostIdProvider, ScriptHostIdProvider>();
             services.TryAddSingleton<IScriptEventManager, ScriptEventManager>();
+
+            // Add Language Worker Services
+            // Need to maintain the order: Add RpcServerInitializationService and then LanguageWorkerChannelInitializationService
```

Comment:
What breaks if this order isn't maintained? Will things fail? Do we need a comment here if the various services are required on construction and will throw if order is changed?

Reply:
RpcServerInitializationService needs to be started first then followed by LanguageWorkerInitializationService before Jobhost initialization. 
Is there a way to add a test for this?